### ISSUE TITLE
Documentation improvment

### ISSUE THREAD

**Issue by MSghais** [AFK-AlignedFamKernel/afk_monorepo#353]

**Comment by Mystic-Nayy** [comment]
Can I take care of this issue?
I am a web3 developer and Technical writer, I will do this effortlessly if i am being assigned, Here are some of the docs and Readme i have written. https://hackmd.io/@xZ4BOZ5TTTy1I0ZQHXFOkg/rkg-YZVCR,
https://hackmd.io/@xZ4BOZ5TTTy1I0ZQHXFOkg/rk8p7TAeJe ,
https://hackmd.io/@xZ4BOZ5TTTy1I0ZQHXFOkg/rkFBcGgAC,
https://github.com/Flex-NFT-Marketplace/Flex-Marketplace-Contract/pull/92/commits/fc28412ba108a61919b75ed1d7ab2e140ce9fbd0,
https://github.com/dragan2234/worldcoin-scroll-bridge/pull/29,

**Comment by MSghais** [comment]
Yes sure assigned thank you! Can you ping me on Telegram so we can discuss it too please?

**Comment by soyaya** [comment]
As a technical product manager, I am confident in my ability to contribute effectively by combining my technical expertise and product management experience. I excel at testing applications, identifying pain points, and creating clear, user-friendly documentation to enhance onboarding and usability. My collaborative approach ensures I can work seamlessly with developers and contributors to drive meaningful improvements.

**Comment by emarc99** [comment]
Is this issue available? I've checked the [docs](https://docs.afk-community.xyz/) and would like to discuss further how to handle this.

**Comment by soyaya** [comment]
> As a technical product manager, I am confident in my ability to contribute effectively by combining my technical expertise and product management experience. I excel at testing applications, identifying pain points, and creating clear, user-friendly documentation to enhance onboarding and usability. My collaborative approach ensures I can work seamlessly with developers and contributors to drive meaningful improvements.

**Comment by nnennaokoye** [comment]
I'd like to take this issue.

**Comment by Imdavyking** [comment]
Can I take this issue?
